resources/js/features/game-list/components/DataTableFacetedFilter/DataTableFacetedFilter.tsx

```diff
@@ -82,14 +82,14 @@ export function DataTableFacetedFilter<TData, TValue>({
           size="sm"
           className={cn(
             'border-dashed',
-            buildTrackingClassNames(`Click ${title} Filter`),
+            buildTrackingClassNames(`Click ${t_title} Filter`),
```

Comment:
For metrics tracking, shouldn't this use an untranslated string?

Reply:
Good call - in latest I've changed this to use the column ID instead.